以下為目前仍有的錯誤訊息紀錄

090706
 l1j.server.server.clientpackets.ClientBasePacket readS
嚴重的: OpCode=66
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
        at java.lang.String.substring(String.java:1938)
        at l1j.server.server.clientpackets.ClientBasePacket.readS(Unknown Source)
        at l1j.server.server.clientpackets.C_Title.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread$HcPacket.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090705 EC編譯時警告訊息
    [javac] Note: E:\workspace\l1j\l1jtw\src\l1j\server\server\model\skill\L1SkillTimer.java uses or overrides a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.

090704
l1j.server.server.ClientThread run
嚴重的: null
java.lang.NullPointerException
        at l1j.server.server.model.Getback.ReadGetbackInfo(Unknown Source)
        at l1j.server.server.model.Getback.GetBack_Location(Unknown Source)
        at l1j.server.server.ClientThread.quitGame(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090703
l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl run
警告: NpcAI發生例外的錯誤。
java.lang.NullPointerException
        at l1j.server.server.model.L1Attack.<init>(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.physicalAttack(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.useSkill(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.skillUse(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.onTarget(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.AIProcess(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.access$400(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090702
l1j.server.server.ClientThread run
嚴重的: null
java.lang.NullPointerException
        at l1j.server.server.clientpackets.C_BoardRead.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090701
l1j.server.server.clientpackets.C_NPCTalk <init>
嚴重的: 找不到對應物件 objid=285290113

090612
l1j.server.server.ClientThread run
Exception in thread "pool-1-thread-8388" java.lang.NullPointerException
        at l1j.server.server.utils.CalcExp.AddExpPet(Unknown Source)
        at l1j.server.server.utils.CalcExp.calcExp(Unknown Source)
        at l1j.server.server.model.Instance.L1MonsterInstance.distributeExpDropKarma(Unknown Source)
        at l1j.server.server.model.Instance.L1MonsterInstance.access$000(Unknown Source)
        at l1j.server.server.model.Instance.L1MonsterInstance$Death.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090611
l1j.server.server.ClientThread run
嚴重的: 21504
java.lang.ArrayIndexOutOfBoundsException: 21504
        at l1j.server.server.model.L1World.getInventory(Unknown Source)
        at l1j.server.server.clientpackets.C_DropItem.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090610
l1j.server.server.ClientThread run
嚴重的: 18432
java.lang.ArrayIndexOutOfBoundsException: 18432
        at l1j.server.server.model.L1World.getInventory(Unknown Source)
        at l1j.server.server.clientpackets.C_DropItem.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090609
l1j.server.server.model.skill.L1SkillUse runSkill
嚴重的: null
java.lang.NullPointerException
        at l1j.server.server.model.skill.L1SkillUse.runSkill(Unknown Source)
        at l1j.server.server.model.skill.L1SkillUse.handleCommands(Unknown Source)
        at l1j.server.server.model.skill.L1SkillUse.handleCommands(Unknown Source)
        at l1j.server.server.clientpackets.C_UseSkill.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread$HcPacket.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090608
l1j.server.server.model.L1PetMatch$L1PetMatchTimer run
警告: null
java.lang.NullPointerException
        at java.util.concurrent.ConcurrentHashMap.containsValue(ConcurrentHashMap.java:799)
        at java.util.concurrent.ConcurrentHashMap.contains(ConcurrentHashMap.java:863)
        at l1j.server.server.model.L1World.getPlayer(Unknown Source)
        at l1j.server.server.model.L1PetMatch.endPetMatch(Unknown Source)
        at l1j.server.server.model.L1PetMatch$L1PetMatchTimer.run(Unknown Source)
        at java.util.TimerThread.mainLoop(Timer.java:512)
        at java.util.TimerThread.run(Timer.java:462)

090607
l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl run
警告: NpcAI發生例外的錯誤。
java.lang.NullPointerException
        at l1j.server.server.model.L1Attack.<init>(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.physicalAttack(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.useSkill(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.skillUse(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.onTarget(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.AIProcess(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.access$400(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090606
l1j.server.server.ClientThread run
嚴重的: null
java.lang.NullPointerException
        at l1j.server.server.clientpackets.C_MoveChar.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)

090605
l1j.server.server.model.skill.L1SkillUse runSkill
嚴重的: Task already scheduled or cancelled
java.lang.IllegalStateException: Task already scheduled or cancelled
        at java.util.Timer.sched(Timer.java:358)
        at java.util.Timer.scheduleAtFixedRate(Timer.java:296)
        at l1j.server.server.model.Instance.L1NpcInstance.startHpRegeneration(Unknown Source)
        at l1j.server.server.model.Instance.L1SummonInstance.setCurrentHp(Unknown Source)
        at l1j.server.server.model.Instance.L1SummonInstance.receiveDamage(Unknown Source)
        at l1j.server.server.model.L1Magic.commitNpc(Unknown Source)
        at l1j.server.server.model.L1Magic.commit(Unknown Source)
        at l1j.server.server.model.skill.L1SkillUse.runSkill(Unknown Source)
        at l1j.server.server.model.skill.L1SkillUse.handleCommands(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.magicAttack(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.useSkill(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.skillUse(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.onTarget(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.AIProcess(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.access$400(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090604
l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl run
警告: NpcAI發生例外的錯誤。
java.lang.NullPointerException
        at l1j.server.server.model.L1Attack.<init>(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.physicalAttack(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.useSkill(Unknown Source)
        at l1j.server.server.model.L1MobSkillUse.skillUse(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.onTarget(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.AIProcess(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance.access$400(Unknown Source)
        at l1j.server.server.model.Instance.L1NpcInstance$NpcAIThreadImpl.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090603
l1j.server.server.clientpackets.ClientBasePacket readS
嚴重的: OpCode=78
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
        at java.lang.String.substring(String.java:1938)
        at l1j.server.server.clientpackets.ClientBasePacket.readS(Unknown Source)
        at l1j.server.server.clientpackets.C_Rank.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread$HcPacket.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090602
l1j.server.server.ClientThread run
嚴重的: null
java.lang.NullPointerException
        at l1j.server.server.clientpackets.C_ItemUSe.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)

090601
l1j.server.server.clientpackets.ClientBasePacket readS
嚴重的: OpCode=92
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
        at java.lang.String.substring(String.java:1938)
        at l1j.server.server.clientpackets.ClientBasePacket.readS(Unknown Source)
        at l1j.server.server.clientpackets.C_ChatWhisper.<init>(Unknown Source)
        at l1j.server.server.PacketHandler.handlePacket(Unknown Source)
        at l1j.server.server.ClientThread$HcPacket.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)
